push/templates/push/applications.html

```diff
@@ -117,7 +117,13 @@
         contentType: 'application/x-www-form-urlencoded; charset=UTF-8'
     });
 
-    api_call.always(function() {
+    api_call.fail(function(jqXHR) {
+      $.each(jqXHR.responseJSON, function(index, element) {
+        $('#id_' + index).after('<div class="alert callout">' + element + '</div>');
+      });
+    });
+
+    api_call.success(function() {
```

Comment:
This changes the `always` redirect to a `success` redirect so the user stays on the page with the form validation errors if there were errors.